test.js

```diff
@@ -1,8 +1,41 @@
 import test from 'ava';
 import cliSpinners from '.';
 
+function mockMathRandom(fixedResult) {
+	unMockMathRandom();
+	const originalImpl = Math.random;
+	Math.random = () => fixedResult;
+	Math.random.originalImpl = originalImpl;
+}
+
+function unMockMathRandom() {
+	if (Math.random.originalImpl) {
+		Math.random = Math.random.originalImpl;
+	}
+}
+
 test('main', t => {
 	t.is(typeof cliSpinners, 'object');
 	t.is(cliSpinners.dots.interval, 80);
 	t.true(Array.isArray(cliSpinners.dots.frames));
 });
+
+test('random getter', t => {
```

Comment:
Not sure what's the preferred style for tests in this repo. Let me know if you think we should structure these tests in a different way. (split the tests cases? move mock methods to a different file? etc)

Reply:
It's good 👍🏻